tests/integration/test_vulnerability_detector/test_feeds/test_validate_feed_content.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import datetime
+import pytest
+
+import wazuh_testing.tools.file as file
+
+
+current_year = datetime.datetime.now().year
```

Comment:
```suggestion
import pytest
import wazuh_testing.tools.file as file

from datetime.datetime import now


current_year = now().year
```

Reply:
Done in 6bf7e6871c48474affeaa53d330e1682285d057f